src/config/chis-tg/config.json

```
@@ -0,0 +1,189 @@
+      "hierarchy": [
+        {
+          "friendly_name": "Sanitary Training",
+          "property_name": "sanitary_training",
+          "contact_type": "cf60_sanitary",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected Supervision Area",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(Supervision Area Name\\)", "\\(SUPERVISION AREA NAME\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "Supervision Area Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "{{ contact.last_name }} {{ contact.first_name }} ({{ lineage.sanitary_training }})",
+          "required": false
+        }
+      ],
+      "contact_properties": [
+        {
+          "friendly_name": "Last Name",
+          "property_name": "last_name",
+          "type": "string",
+          "parameter": ["\\(LAST NAME\\)", "\\(LASTNAME\\)", "\\(LAST-NAME\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "First Name",
+          "property_name": "first_name",
+          "type": "string",
+          "parameter": ["\\(Chw First Name\\)", "\\(Chw Firstname\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "Full Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "{{ contact.last_name }} {{ contact.first_name }}",
+          "required": false
+        },
+        {
+          "friendly_name": "Gender",
+          "property_name": "sex",
+          "type": "gender",
+          "required": true
+        },
+        {
+          "friendly_name": "Telephone",
+          "property_name": "telephone",
+          "type": "phone",
+          "parameter": "TG",
+          "required": true
+        },
+        {
+          "friendly_name": "NGO/Provenance Structure",
+          "property_name": "appartenance_ngo",
+          "type": "string",
+          "parameter": ["\\(STRUCTURE[ -]NAME\\)"],
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "ci90_chw_site",
+      "friendly": "CHW Post",
+      "contact_type": "ci92_chw_site",
+      "user_role": ["chw", "community_user"],
+      "username_from_place": true,
+      "deactivate_users_on_replace": true,
+      "hierarchy": [
+        {
+          "friendly_name": "Village",
+          "property_name": "village",
+          "contact_type": "ch80_village",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected CHW Post",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(Chw Post Name\\)", "\\(CHW POST NAME\\)", "\\(POST NAME\\)", "\\(Post Name\\)"],
```

```suggestion
        "parameter": ["\\(*Post*\\)""],
```

Remove anything in brackets including `post`?